.travis.yml

```diff
@@ -58,23 +58,3 @@ jobs:
       script:
         - gem install brakeman
         - brakeman --run-all-checks
-
-  allow_failures:
-
-    - stage: test
-      name: 'yarn audit'
-      script:
-        - yarn audit
-
-    - stage: test
-      name: 'bundle audit'
-      script:
-        - gem install bundler-audit
-        - bundle audit update
-        - bundle audit check
-
-    - stage: test
-      name: 'brakeman check'
-      script:
-        - gem install brakeman
-        - brakeman --run-all-checks
```

Comment:
These tests are still in `.travis.yml`. Removing these lines just made them required.